tests/sklearn_test_utils.py

```diff
@@ -0,0 +1,88 @@
+# tests.sklearn_test_utils
```

Comment:
Had to copy these functions over because sklearn uses nose as part of their testing framework. As we no longer have that as a dependency, it was breaking on import

Reply:
Nice - I think this was definitely the right decision in the face of nose dependencies. I looked through the code in `assert_warns_message` and I don't necessarily see anything that is different from [`pytest.warns`](https://docs.pytest.org/en/latest/warnings.html#warns). 

Would it just be easier to use [`pytest.deprecated_call`](https://docs.pytest.org/en/latest/warnings.html#ensuring-function-triggers)?